pod-operation/src/components/lim_temperature.rs

```diff
@@ -32,8 +36,11 @@ pub struct LimTemperature {
 
 impl LimTemperature {
 	pub fn new(device_address: SlaveAddr) -> Self {
+		sleep(Duration::from_secs(1));
```

Comment:
Question: is this sleep time still needed?

Reply:
lol nope thx